src/index.tsx

```diff
@@ -5,18 +5,18 @@ import { RoughCanvas } from "roughjs/bin/canvas";
 
 import "./styles.css";
 
-type ExcaliburElement = ReturnType<typeof newElement>;
-type ExcaliburTextElement = ExcaliburElement & {
+type ExcalidrawElement = ReturnType<typeof newElement>;
+type ExcaliburTextElement = ExcalidrawElement & {
```

Comment:
ExcaliburTextElement can also be renamed

Reply:
Done!